content/spin/v2/rdbms-storage.md

```diff
@@ -74,8 +80,19 @@ The Python SDK doesn't currently surface the MySQL or PostgreSQL APIs.
 
 {{ startTab "TinyGo"}}
 
-The Go SDK doesn't currently surface the MySQL or PostgreSQL APIs.
+MySQL functions are available in the `github.com/fermyon/spin/sdk/go/v2/mysql` package, and PostgreSQL in `github.com/fermyon/spin/sdk/go/v2/pg`. [See Go Packages for reference documentation.](https://pkg.go.dev/github.com/fermyon/spin/sdk/go/v2).
```

Comment:
```suggestion
MySQL functions are available in the `github.com/fermyon/spin/sdk/go/v2/mysql` package, and PostgreSQL in `github.com/fermyon/spin/sdk/go/v2/pg`. [See Go Packages for reference documentation](https://pkg.go.dev/github.com/fermyon/spin/sdk/go/v2).
```

(remove the extra period in last sentence.)

Reply:
Oof, thanks for the comment.  I was gazing at the diff like the person in the "Corporate needs you to find the difference between these two pictures" meme.

Reply:
IT'S EVERYWHERE.  Although not any more. I hope.